marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/data-services/entitySearch/getOpticPlanTest.sjs

```diff
@@ -0,0 +1,89 @@
+const search = require('/MarkLogic/appservices/search/search');
+const test = require("/test/test-helper.xqy");
+const op = require('/MarkLogic/optic');
+
+const queryOptions = '<options xmlns="http://marklogic.com/appservices/search"/>';
+
+function invokeOpticPlanService(schemaName, viewName, limit, structuredQuery, searchText, sortOrder, columns, retries=0) {
+    try {
+        // Need to clear the modules cache or unrelated error could be thrown. Investigating to file a bugtrack.
+        xdmp.moduleCacheClear();
```

Comment:
Does this need to be done on every iteration of the try/sleep loop?

Reply:
I'll change this to when retries = 0.